moveit_reach_plugins/src/evaluation/manipulability_moveit.cpp

```diff
@@ -141,9 +169,93 @@ double ManipulabilityRatio::calculateScore(const Eigen::MatrixXd& jacobian_singu
   return jacobian_singular_values.minCoeff() / jacobian_singular_values.maxCoeff();
 }
 
+bool ManipulabilityNormalized::initialize(XmlRpc::XmlRpcValue& config)
+{
+  bool ret = ManipulabilityMoveIt::initialize(config);
+
+  if (config.hasMember("exclusion_list") && config["exclusion_list"].getType() == XmlRpc::XmlRpcValue::TypeArray)
+  {
+    for (std::size_t i = 0; i < config["exclusion_list"].size(); ++i)
+    {
+      std::string excluded_link = static_cast<std::string>(config["exclusion_list"][i]);
+      exclusion_list_.push_back(excluded_link);
+    }
```

Comment:
Probably helpful to define the parameter string once and use a variable to avoid misspelling and make it easier to change in future if necessary
```suggestion
  const std::string exclusion_list_param = "exclusion_list";
  if (config.hasMember(exclusion_list_param) && config[exclusion_list_param].getType() == XmlRpc::XmlRpcValue::TypeArray)
  {
    for (std::size_t i = 0; i < config[exclusion_list_param].size(); ++i)
    {
      std::string excluded_link = static_cast<std::string>(config["exclusion_list"][i]);
      exclusion_list_.push_back(excluded_link);
    }
```

Reply:
Also, I think we should make the name `excluded_links` to be more specific that the entries in this list are links of the URDF that will be excluded from the calculation of the characteristic length